pkg/deps/config/suppliers.go

```diff
@@ -743,6 +743,7 @@ func NewSupplyRelayAuthenticatorFn(
 //   - SupplierFn: Supplier function for dependency injection
 func NewSupplyRelayerProxyFn(
 	servicesConfigMap map[string]*relayerconfig.RelayMinerServerConfig,
+	pingEnabled bool,
```

Comment:
Do one of the following:
- Either add this to the `Parameters` section in the comment above 
- Remove `Parameters` from the comment altogether

My bias is for the latter.